workers/setup.cfg

```diff
@@ -2,15 +2,17 @@
 # Read more about `setup.cfg`:
 # https://docs.python.org/3/distutils/configfile.html
 
-
 [flake8]
 # W503: https://github.com/python/black#line-breaks--binary-operators
 # E203: https://github.com/psf/black#slices
 # These settings are compatible with Black
 ignore = W503, E203
 inline-quotes = double
 max-line-length = 88
-
+exclude =
+per-file-ignores =
+    ohsome_quality_analyst/indicators/mapping_saturation/*:N8
+    ohsome_quality_analyst/api.py:N803,N815
```

Comment:
* `N803`: What is the reason we can't use lowercase argument names here?
* `N815`: Same question for variable names in class scope.

Reply:
Those variable names are used by FastAPI to generate the API parameters. I am not sure about the reason why but in the beginning those parameter where definied as mixed case.

If we change those variable names it would also change the API entpoint parameters.

Reply:
I will add a note in this config file